playbooks/generic-deploy_airship.yml

```diff
@@ -2,8 +2,19 @@
 - hosts: soc-deployer
   gather_facts: yes
   any_errors_fatal: true
+  pre_tasks:
+    - name: Load generic vars
+      include_vars: "{{ item }}"
+      loop:
+        - "{{ playbook_dir }}/../vars/common-vars.yml"
+        - "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
```

Comment:
Not clear if there is any specific purpose of using vars as pre_tasks. If its just vars availability across roles, may be  vars_files will be more concise ?  https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#defining-variables-in-files

Reply:
Fixed. Thanks!